### DISCUSSION TITLE
Allow wysiwyg interface for nvarchar field

### DISCUSSION THREAD

**Discussion by devistar** [directus/directus#9106] [Feature Requests]
It would be nice to allow the wysiwyg interface to be used with the nvarchar fields.

This allows us to set a limit for the text.

Additionally in the use case of SQL Server, text columns are deprecated.

"ntext, text, and image data types will be removed in a future version of SQL Server. Avoid using these data types in new development work, and plan to modify applications that currently use them. Use nvarchar(max), varchar(max), and varbinary(max) instead."

**Comment by rijkvanzanten** [comment]
I think this might be as simple as treating `nvarchar` as a "text" type in Directus, eg in here:

https://github.com/directus/directus/blob/4e7c6a1d4570cde6e4494c24e3b863d0d08d2bf3/api/src/utils/get-local-type.ts#L30

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
